llm_studio/src/utils/data_utils.py

```diff
@@ -149,6 +149,7 @@ def read_dataframe_drop_missing_labels(
     df[input_cols] = df[input_cols].fillna("").astype(str)
     if (
         hasattr(cfg.dataset, "answer_column")
+        and not isinstance(cfg.dataset.answer_column, list)
```

Comment:
Why list and not tuple? I think we need to be more strict with the types here so be sure we don't miss a case

Reply:
because we cast it to list at some point, we can make it a check for tuple and list to make sure